### ISSUE TITLE
Spec file changelog contains bogus date  (generate warning)

### ISSUE THREAD

**Issue by arekkusu** [ltb-project/openldap-rpm#17]
When building the RPM  following warning occur due to bogus dates in spec changelog.
(day of the week does not match date).

```
    default: warning: bogus date in %changelog: Wed May 05 2017 - Clement Oudot <clem@ltb-project.org> - 2.4.44-3
    default: warning: bogus date in %changelog: Wed Apr 02 2013 - Clement Oudot <clem@ltb-project.org> - 2.4.35-1 / 1.1-8
    default: warning: bogus date in %changelog: Thu Mar 12 2013 - Clement Oudot <clem@ltb-project.org> - 2.4.34-1 / 1.1-8
    default: warning: bogus date in %changelog: Thu Apr 24 2012 - Clement Oudot <clem@ltb-project.org> - 2.4.31-1 / 1.1-8
    default: warning: bogus date in %changelog: Tue Apr 29 2009 - Clement Oudot <clem@ltb-project.org> - 2.4.16-1 / 1.0.3-4
    default: warning: bogus date in %changelog: Fri Jan 15 2009 - Clement Oudot <clement.oudot@linagora.com> - 2.4.13-1
    default: warning: bogus date in %changelog: Mon Oct 6 2005 - Raphael Ouazana <raphael.ouazana@linagora.com> - 2.2.28-4.2
    default: warning: bogus date in %changelog: Mon Oct 6 2005 - Raphael Ouazana <raphael.ouazana@linagora.com> - 2.2.28-4
    default: warning: bogus date in %changelog: Thu Aug 30 2005 - Clement Oudot <clement.oudot@linagora.com> - 2.2.28-1
```

**Comment by arekkusu** [comment]
Let me fix this ;)

**Comment by coudot** [comment]
Fixed by #19
